test/forge/CarbonController.sol

```diff
@@ -0,0 +1,183 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.19;
```

Comment:
Perhaps `CarbonController.t.sol`?

Reply:
Replaced all forge test filenames to ones with the same format.